static/js/components/search/CustomNoHits.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+import React from 'react';
+import { NoHits } from 'searchkit';
+
+export default class CustomNoHits extends NoHits {
+  render() {
+    let message;
+
+    if ((this.hasHits() || this.isInitialLoading() || this.isLoading()) && !this.getError()) {
+      return null;
+    }
+
+    if (this.getError()) {
+      if (this.getError().data && this.getError().data.error) {
+        message = this.getError().data.error;
```

Comment:
This is great! However, while I was manually testing this, I discovered that Django Rest Framework sets the error description in a property called `detail`, not a property called `error`. As a result, if I try the search as a user that is forbidden to access the search API due to lack of permissions, I get this AJAX response:
![error-message](https://cloud.githubusercontent.com/assets/132355/19891688/9cf41924-a017-11e6-9ddc-08f40b1e3304.png)
But the UI displays the generic error message:
![generic-error](https://cloud.githubusercontent.com/assets/132355/19891692/a0c4bb6c-a017-11e6-948d-007ac5abeeac.png)

Could we modify the error response in `search/views.py` to use the `detail` property, and then make this Javascript check that _instead_ of the `error` property? That way, the detailed error message will show up in either case.